### PR TITLE
Added Cisco 6500 and implemented verify unicast command

### PR DESCRIPTION
**Pull request by Marx314**
We separated the two code path since not all cisco switches support this command

### CODE REVIEW

File: fake_switches/cisco/command_processor/config_interface.py [hunk 1]
```diff
@@ -128,6 +128,8 @@ def do_ip(self, *args):
             self.port.ip_redirect = True
         if "proxy-arp".startswith(args[0]):
             self.port.ip_proxy_arp = True
+        if "verify unicast".startswith(args[0]) and isinstance(self.port, VlanPort):
```
Comment (JoProvost) [inline fake_switches/cisco/command_processor/config_interface.py]: I'm pretty sure that we split on space and that arg[0] will be "verify" and arg[1] will be "unicast".  So "verify multicast" for example will still fall in this if statement.

File: fake_switches/cisco/command_processor/config_interface.py [hunk 2]
```diff
@@ -166,7 +168,8 @@ def do_no_ip(self, *args):
                 self.port.vrf = None
         if "redirects".startswith(args[0]):
             self.port.ip_redirect = False
-
+        if "verify unicast".startswith(args[0]) and isinstance(self.port, VlanPort):
```
Comment (JoProvost) [inline fake_switches/cisco/command_processor/config_interface.py]: I'm pretty sure that we split on space and that arg[0] will be "verify" and arg[1] will be "unicast".  So "verify multicast" for example will still fall in this if statement.

File: fake_switches/cisco/command_processor/config_interface.py (hunk 1 above)
Comment (lindycoder) [inline fake_switches/cisco/command_processor/config_interface.py]: that's correct

**Comment by Marx314** [comment]
@JoProvost @fbouliane ^^

File: fake_switches/cisco/cisco_core.py [hunk 3]
```diff
@@ -26,10 +26,9 @@
 from fake_switches.switch_configuration import Port
 from fake_switches.terminal import LoggingTerminalController
```
Comment (lindycoder) [inline fake_switches/cisco/cisco_core.py]: 👍

**Pull request merged**